microtiter_plates/libraries/platelayoutgenerator/source.rb

```diff
@@ -35,6 +42,16 @@ def next_group(column: nil)
     @layout.slice!(i, @group_size)
   end
 
+  def iterate_column(column)
+    return nil if column.nil?
```

Comment:
@column is initialized as []. Is there anywhere it can be set to nil?

Reply:
Use `column.blank?` instead. (I think this is a Rails method, so may not show up in the Ruby API)